package.json

```diff
@@ -9,7 +9,7 @@
     "test": "mocha"
   },
   "dependencies": {
-    "source-map": "0.1.32"
+    "source-map": "^0.1.33"
```

Comment:
I was just being conservative - that was the first version that needed the changes.

But you're probably right that it should just be bumped all the way up, because a caret version will only match the second digit while the first one is zero.